Billmike/More-Recipes
# Refactored Backend

# **What does this PR do?**

Refactor Backend Dependencies for More-Recipes project

# **Description of Task to be completed?**

Setup Nodejs
Setup Sequelize
Setup Expressjs
Setup PostgreSQL
# **How should this be tested?**

After cloning remote repo, cd into server folder
run npm run start:dev
Navigate to localhost:8000 in your favorite browser to see the result.

# **What are the relevant pivotal tracker stories?**

#150670064
